app/cinema/hall.py

```diff
@@ -0,0 +1,21 @@
+from app.people.customer import Customer
+
+from app.people.cinema_staff import Cleaner
+
+
+class CinemaHall:
+
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(
+            self,
+            movie_name: str,
+            customers: dict[Customer],
```

Comment:
`customers` variable should indicate that it is a list of `Customer` class instances, not dict.  
```suggestion
            customers: list[Customer],
```